networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.adoc

```diff
@@ -57,6 +57,7 @@ include::modules/nw-ingress-sharding-route-configuration.adoc[leveloffset=+1]
 
 * The Ingress Operator manages wildcard DNS. For more information, see
```

Comment:
The sentence format here needs to be changed to a list format, please: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

I missed this in my initial review.

Reply:
Ack.